src/assets/theme.ts

```diff
@@ -0,0 +1,18 @@
+export default {
+  flexboxgrid: {
+    // Defaults
+    gutterWidth: 3, // rem
+    outerMargin: 2, // rem
+    container: {
+      sm: 46, // rem
+      md: 61, // rem
+      lg: 76,  // rem
+    },
+    breakpoints: {
+      xs: 0,
+      sm: 768,
+      md: 1024,
+      lg: 1200,
+    },
+  },
+}
```

Comment:
Abbreviations are a bit of a pet peeve of mine as they introduce ambiguity to new readers for no real good reason. Can you change these to long form?